### ISSUE TITLE
Not compatible with the version of your running IDE (PhpStorm 2022.3.3)

### ISSUE THREAD

**Issue by sprankhub** [kalessil/phpinspectionsea#1881]
The title says it all. https://plugins.jetbrains.com/plugin/16935-php-inspections-ea-ultimate- shows:

![image](https://user-images.githubusercontent.com/930199/226212204-ddff328d-6946-4c9b-a103-9f12646c5e4a.png)

**Comment by elomonaco** [comment]
@kalessil please fix this

**Comment by elomonaco** [comment]
I've pretty much abandoned this plugin, I'm really sad this plugin just died, it's really good and it's why I spent money to ensure it stayed alive, but it seems like the project lost all momentum. If the dev ever comes back I'll happily support it again but right now I've moved onto PHP Hammer since it's the new best thing in this space (this one is still far better, but hard to use something that you can't even install anymore and at least the other  plugin is actively developed on)

**Comment by kaliberx** [comment]
@kalessil Please fix this bug immediately! This is a critical bug. We pay you money for a product which doesn't work on the latest PhpStorm! Shame on you!

**Comment by kalessil** [comment]
@ea-inspections-team, what's going on? Turns out the plugins are unmaintained for several months. Sorry for making this public, but that's unacceptable!

**Comment by kalessil** [comment]
FYI, folks:  https://github.com/kalessil/phpinspectionsea/issues/1882

**Comment by elomonaco** [comment]
Glad to hear someone to trying to keep things moving, I appreciate that @kalessil I hope we start to see some progress soon as I really do like this plugin and hate seeing it in the state its in.

**Comment by janklan** [comment]
@kalessil I suspect you've taken more of an IP holder role with this plugin and your comment above suggests you outsource its development to others. 

I suggest to contact @Haehnchen who is in charge of a bunch of other PHP-related plugins (https://plugins.jetbrains.com/organizations/espend_de). 

He brought the Symfony Support plugin back to life, started charging a few dollars (with 6m+ downloads I hope he's loaded now!) and everything works great - no more bugs and improvements are pouring in. 

In other words, I can vouch for him. I wonder how many others would give this post a thumbs up to indicate the would vouch with me.

Maybe he would be interested in helping out? 

@Haehnchen FYI.

**Comment by kalessil** [comment]
> I suspect you've taken more of an IP holder role with this plugin and your comment above suggests you outsource its development to others.

@janklan : something like that, mainly I handed over both EA/Ultimate plugins to folks and pulled myself from development and financial aspects. Though I have the power to initiate a transfer of the EA-team brand and ownership to a backup team, both teams agreed on the terms and needed expertise.

Daniel is a great maintainer, agree! Though before I reach more folks, I want to sort out things with the EA Team - there is a clear path forward defined for emergency cases, when they can not perform due to e.g. military actions in their location.

**Comment by Haehnchen** [comment]
thanks for ping me. funnily: i was looking the issues this morning, because noticed missing really important inspections for me.
fyi: i just noticed the "owner ship" transfer for the project, but i am monitoring the project since then.

hopefully it will be sorted out.

**Comment by elomonaco** [comment]
hopefully this gets pushed up soon, I'd love to be able to use this plugin again if possible :)

**Comment by janklan** [comment]
@kalessil any updates?

**Comment by elomonaco** [comment]
this should be completed finally @kalessil you should be good to close this issue.
